clients/tabby-chat-panel/src/index.ts

```diff
@@ -37,12 +37,13 @@ export interface ServerApi {
   sendMessage: (message: ChatMessage) => void
   showError: (error: ErrorMessage) => void
   cleanError: () => void
+  addClientSelectedContext: (context: Context) => void
```

Comment:
```suggestion
  addRelevantContext: (context: Context) => void
```

To match field name in line 53